### ISSUE TITLE
No imagery list in the Backgrounds section

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/iD#7966]
I've just run a new version of id 2.18.5 and there is no background imagery. They appeared after refreshing F5.

![imagery](https://user-images.githubusercontent.com/57260466/92655849-4e6d1300-f2f2-11ea-8118-96c6ff4f8894.png)

**Comment by quincylvania** [comment]
@maro-21 Have you seen this more than once? It could have been a network issue since iD loads the background source data asynchronously now.

**Comment by maro-21** [comment]
Yes, I used to have it in the past a few times

**Comment by quincylvania** [comment]
@maro-21 But are you still seeing it? Is there a location you can reliably reproduce it?

**Comment by JamesKingdom** [comment]
Might still be a good idea to add some sort of error handling if the imagery fails to load?

**Comment by maro-21** [comment]
No, I had it only once in this version. I remember that I had it maybe twice in the past as well.

**Comment by Dimitar5555** [comment]
I've also had this issue from time to time. Usually collapsing and expanding the section with imageries fixes the problem. It seems to happen randomly,

**Comment by k-yle** [comment]
this happens to me occasionally too – [the json file from ELI](https://cdn.jsdelivr.net/gh/osmlab/editor-layer-index@gh-pages/imagery.geojson) takes >30 seconds to download, and/or the browser eventually gives up and errors.

I think we could add a loading indicator and an error message with a retry button. Something like this maybe:

<img width="300" alt="Image" src="https://github.com/user-attachments/assets/64d88412-46f9-42f9-b7e8-29e942826cf6" />

<img width="300" alt="Image" src="https://github.com/user-attachments/assets/9435c34b-fcc8-4f43-b5ca-372afaefb895" />

hopefully most users would never see this UI

**Comment by maro-21** [comment]
I'd like to mention that I haven't seen this error since my post in 2020. But since it still occurs, I will keep the issue open.


> I think we could add a loading indicator and an error message with a retry button. Something like this maybe:

The one on the right is definitely better.
